### PR TITLE
Base+Browser+LibWeb+WebContent: Add ability to inspect session storage

### PR DESCRIPTION
**Pull request by Sauler**
This PR adds ability to inspect session storage in Storage Inspector window.
I also added test page for Web Storage API.
image

image

### CODE REVIEW

File: Base/res/html/misc/storage.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        const value = storage.getItem(key);
+        if (!value) {
+            document.getElementById(resultID).innerHTML = "No value found for '" + key + "'";
+            return;
+        }
+
+        document.getElementById(resultID).innerHTML = "Value for key '" + key + "' = " + storage.getItem(key);
+    }
+</script>
+</html>
```
Comment (linusg) [inline Base/res/html/misc/storage.html:128]: Please configure your editor to add a newline at the end of files

**Review by linusg — CHANGES_REQUESTED** [review]

File: Base/res/html/misc/storage.html (hunk 1 above)
Comment (Sauler) [reply Base/res/html/misc/storage.html]: For some reason i can't fix this. I added new line in this file, amend last commit and force pushed but github still does not show this change.

File: Base/res/html/misc/storage.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
```
Comment (linusg) [inline Base/res/html/misc/storage.html:129]: This now has two newlines, that's not what I meant either - and in fact, it fails the linter :)

```
bad:  text
good: text\n
bad:  text\n\n
```

**Review by linusg — CHANGES_REQUESTED** [review]

**Comment by Sauler** [comment]
@linusg can you check it again?

**Pull request merged**